package.json

```diff
@@ -33,7 +33,7 @@
     "pkg-all": "source .circleci/local_publish_helpers.sh && generatePkgCli",
     "pkg-all-local": "yarn verdaccio-start && yarn verdaccio-connect && yarn publish-to-verdaccio && yarn pkg-all && yarn verdaccio-disconnect",
     "publish:master": "lerna publish --canary --force-publish --preid=alpha --exact --include-merged-tags --conventional-prerelease --no-verify-access --yes",
-    "publish:dev": "lerna publish --canary --force-publish --preid=alpha --exact --include-merged-tags --conventional-prerelease --no-verify-access --yes",
+    "publish:dev": "lerna publish --preid=rc+$(git rev-parse --short HEAD) --exact --include-merged-tags --conventional-prerelease --no-verify-access --yes --conventional-commits --dist-tag rc",
```

Comment:
have we checked to make sure an exact version with a `+` can be installed from npm? The semver spec says anything after the `+` can be ignored, just not sure what the exact npm implementation is